### PR TITLE
Prl 1460 latest

### PR DESCRIPTION
**Pull request by swaroopa86**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-1460

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/respondent/respondentcaseSequence.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import { YesOrNo } from 'app/case/definition';
+import { Sections, Step } from '../constants';
+import { DETAILS_KNOWN, RESPONDENT_TASK_LIST_URL, START_ALTERNATIVE, MIAM_START, MIAM_ATTEND_WILLINGNESS, PRIVATE_DETAILS_CONFIRMED, PRIVATE_DETAILS_NOT_CONFIRMED, MIAM_SUMMARY, CHECK_ANSWERS, SAFETY_MAIN_PAGE, YOUR_SAFETY, DOMESTIC_ABUSE_RISK, DOMESTIC_ABUSE_RISK_NO } from '../urls';
```
Comment (cshiwarale) [inline src/main/steps/respondent/respondentcaseSequence.ts:3]: Duplicate imports, causing lint errors

Comment (swaroopa86) [reply src/main/steps/respondent/respondentcaseSequence.ts:3]: this happened a few mins ago, while merging the changes from base branch. I am resolving it

File: src/main/steps/respondent/task-list/utils.ts [hunk 2]
```diff
@@ -20,3 +20,14 @@ export const getMiamStatus = (userCase: CaseWithId): SectionStatus => {
   }
   return SectionStatus.NOT_STARTED;
 };
+
+
+export const getYourSafetyStatus = (userCase: CaseWithId): SectionStatus => {
+  if (userCase?.detailsKnown ) {
```
Comment (cshiwarale) [inline src/main/steps/respondent/task-list/utils.ts:26]: Is this correct condition check for 'your safety' task?

File: src/main/steps/respondent/task-list/utils.ts [hunk 3]
```diff
@@ -20,3 +20,13 @@ export const getMiamStatus = (userCase: CaseWithId): SectionStatus => {
   }
   return SectionStatus.NOT_STARTED;
 };
+
+export const getYourSafetyStatus = (userCase: CaseWithId): SectionStatus => {
+  if (userCase?.safetyConcerns) {
+    return SectionStatus.COMPLETED;
+  }
+  // if (userCase?.detailsKnown ) {
```
Comment (cshiwarale) [inline src/main/steps/respondent/task-list/utils.ts:28]: Please remove commented code.

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**